tests/gold_tests/remap/remap_http.test.py

```diff
@@ -28,7 +28,7 @@
 # Define default ATS
 ts = Test.MakeATSProcess("ts")
 server = Test.MakeOriginServer("server")
-dns = Test.MakeDNServer("dns")
+dns = Test.MakeDNServer("dns", default=["127.0.0.1"])
```

Comment:
If we don't need this for the test we should not add it

Reply:
done